src/Cli/Microsoft.TemplateEngine.Cli/Commands/xlf/SymbolStrings.pl.xlf

```diff
@@ -107,6 +107,11 @@ Jeśli polecenie zostanie określone bez argumentu, zostanie wyświetlona lista
         <target state="translated">Wyszukuje tylko aktualizacje i wyświetla pakiety szablonów do zaktualizowania bez stosowania aktualizacji.</target>
         <note />
       </trans-unit>
+      <trans-unit id="ListCommand_Option_Format">
+        <source>Specifies the output format type for the list packages command</source>
```

Comment:
Określa format w jakim zostanie zwrócony wynik wywołania komendy list packages

Reply:
Hey - unfortunately we cannot accept translations for any localizable text in this project, they all have to go through our internal localization process. We have some documentation about it here: https://github.com/dotnet/sdk/blob/main/documentation/project-docs/Localization.md

What's been done in this PR (defining new strings) is totally fine, but we can't do anything more than that.

Reply:
Hi, no worries - I rather added it as a proposal or hint or something like that. I am currently on a stage of setting this repository to be debuggable on my PC locally, but many thanks for the additional md resource